addons/website/models/website.py

```diff
@@ -1811,20 +1818,28 @@ def _trigram_enumerate_words(self, search_details, search, limit):
                         search=unaccent(sql.Placeholder('search')),
                         field=unaccent(sql.SQL('value'))
                     )
+                    where_clause_domain = """WHERE lang = {lang}
+                                           AND name = ANY({names})
+                                           AND type = 'model'
+                                           AND value IS NOT NULL
+                                           """
```

Comment:
Nitpicking but will help should we add another (long) condition here, to not waste space for no reason
```suggestion
                    where_clause_domain = """
                        WHERE lang = {lang}
                        AND name = ANY({names})
                        AND type = 'model'
                        AND value IS NOT NULL
                    """
```